HPXMLtoOpenStudio/measure.rb

```diff
@@ -1714,18 +1715,12 @@ def self.add_heat_pump(runner, model, weather, spaces, airloop_map)
                                                                      sequential_heat_load_fracs, sequential_cool_load_fracs,
                                                                      living_zone)
 
-      elsif [HPXML::HVACTypeHeatPumpAirToAir].include? heat_pump.heat_pump_type
-
-        airloop_map[sys_id] = HVAC.apply_central_air_to_air_heat_pump(model, runner, heat_pump,
-                                                                      sequential_heat_load_fracs, sequential_cool_load_fracs,
-                                                                      living_zone)
-
-      elsif [HPXML::HVACTypeHeatPumpMiniSplit].include? heat_pump.heat_pump_type
-
-        airloop_map[sys_id] = HVAC.apply_mini_split_heat_pump(model, runner, heat_pump,
-                                                              sequential_heat_load_fracs, sequential_cool_load_fracs,
-                                                              living_zone)
-
+      elsif [HPXML::HVACTypeHeatPumpAirToAir,
+             HPXML::HVACTypeHeatPumpMiniSplit,
+             HPXML::HVACTypeHeatPumpPTHP].include? heat_pump.heat_pump_type
+        airloop_map[sys_id] = HVAC.apply_central_air_source_hvac_systems(model, runner, heat_pump, heat_pump,
```

Comment:
I think I suggested this method name, but now I'm wondering why I included "central" in the name considering we're handling room ACs, etc. Perhaps we should just use "apply_air_source_hvac_systems"?